meta-ostro-xt/recipes-devtools/checksec/checksec_1.5-1.bbappend

```diff
@@ -0,0 +1,3 @@
+SRC_URI[md5sum] = "075996be339ab16ad7b94d6de3ee07bd"
+SRC_URI[sha256sum] = "77b8a7fd9393d10def665658a41176ee745d5c7969a4a0f43cefcc8a4cd90947"
```

Comment:
@mythi  Yes, bitbake complained abt the mismatch.

ERROR: checksec-1.5-1-r0 do_fetch: Checksum failure fetching http://www.trapkit.de/tools/checksec.sh
ERROR: checksec-1.5-1-r0 do_fetch: Function failed: Fetcher failure for URL: 'http://www.trapkit.de/tools/checksec.sh'. Checksum mismatch!
File: '/var/tmp/ostro-os-xt/bb-cache/sources/checksec.sh' has md5 checksum 075996be339ab16ad7b94d6de3ee07bd when 57cc3fbbbe48e8ebd4672c569954374d was expected
File: '/var/tmp/ostro-os-xt/bb-cache/sources/checksec.sh' has sha256 checksum 77b8a7fd9393d10def665658a41176ee745d5c7969a4a0f43cefcc8a4cd90947 when 05822cd8668589038d20650faa0e56f740911d8ad06f7005b3d12a5c76591b90 was expected
If this change is expected (e.g. you have upgraded to a new version without updating the checksums) then you can use these lines within the recipe:
SRC_URI[md5sum] = "075996be339ab16ad7b94d6de3ee07bd"
SRC_URI[sha256sum] = "77b8a7fd9393d10def665658a41176ee745d5c7969a4a0f43cefcc8a4cd90947"